container/state.go

```diff
@@ -268,13 +268,23 @@ func (s *State) SetExitCode(ec int) {
 	s.ExitCodeValue = ec
 }
 
+// SetRunning sets the running state along with StartedAt time.
+func (s *State) SetRunning(ctr libcontainerdtypes.Container, tsk libcontainerdtypes.Task, start time.Time) {
+	s.setRunning(ctr, tsk, &start)
+}
+
+// SetRunningExternal is sets the running state without setting the `StartedAt` time (used for containers not started by Docker instead of SetRunning).
```

Comment:
```suggestion
// SetRunningExternal sets the running state without setting the `StartedAt` time (used for containers not started by Docker instead of SetRunning).
```